components/hierarchical-view/hierarchical-view-mixin.js

```diff
@@ -14,15 +14,16 @@ export const HierarchicalViewMixin = superclass => class extends superclass {
 			/**
 			 * @ignore
 			 */
-			childView: { type: Boolean, reflect: true, attribute: 'child-view' },
+			hierarchicalView: { type: Boolean },
 			/**
 			 * @ignore
 			 */
-			hierarchicalView: { type: Boolean },
+			rootView: { type: Boolean, attribute: 'root-view' },
 			/**
 			 * @ignore
 			 */
-			shown: { type: Boolean, reflect: true }
+			shown: { type: Boolean, reflect: true },
+			_childView: { type: Boolean, reflect: true, attribute: 'child-view' },
```

Comment:
I think in other cases we similarly apply the underscore to the reflected property (`_child-view`). It's not a big deal, but when we see the attribute in the dev console it becomes extra clear that's it has been reflected internally.

Reply:
It is used in the LMS code [here](https://search.d2l.dev/xref/lms/lp/framework/web/D2L.LP.Web.UI/Desktop/Controls/Menu/ClassInitializer.js?r=5264bcc5&mo=109&fi=4#4) so we'd want to update that somewhat semi-simultaneously if we want to make that change.

Reply:
Gotcha. Ok, maybe not worth it depending on your thoughts on my other comment (i.e. consumers would be able to use the public `rootView` instead, setting us up to eventually remove `childView`).